tfx_bsl/public/beam/run_inference.py

```diff
@@ -44,19 +53,84 @@ def RunInference(  # pylint: disable=invalid-name
      `ai_platform_prediction_model_spec` field is set in
      `inference_spec_type`.
 
-  TODO(b/131873699): Add support for the following features:
-  1. Bytes as Input.
-  2. PTable Input.
-  3. Models as SideInput.
-
```

Comment:
Why remove this?

Reply:
Isn't harrison Working on this? 
I can add it back though